libraries/AP_Scripting/examples/6DoF_roll_pitch.lua

```diff
@@ -0,0 +1,54 @@
+-- This script sets the target roll and pitch to the current value when arming or when E-stop is removed
```

Comment:
We should expand these comments a lot more maybe even provide a link to a wiki page where the 6DOF copter setup is explained.

I think ideally users should be able to look at this description and understand how the script should be used and its purpose.  So for example, a normal user might look at this and think it is useful in some way for a regular multicopter.